Library/Formula/hugo.rb

```diff
@@ -195,8 +175,10 @@ def install
     ln_sf buildpath, buildpath/"src/github.com/spf13/hugo"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "go", "build", "main.go"
-    bin.install "main" => "hugo"
+    system "go", "build", "-o", bin/"hugo", "main.go"
+
+    bash_completion.mkpath
+    system bin/"hugo", "genautocomplete", "--completionfile=#{bash_completion}/hugo.sh"
```

Comment:
@dunn You probably want to use the `bash_completion.install` and generate it into the buildpath instead. Can you check if the script contains any hardcoded references to the full path? Thanks!